src/main/java/thrift/logic/commands/RedoCommand.java

```diff
@@ -0,0 +1,32 @@
+package thrift.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import thrift.logic.commands.exceptions.CommandException;
+import thrift.model.Model;
+
+/**
+ * Redo undone command in THRIFT.
+ */
+public class RedoCommand extends Command {
+
+    public static final String COMMAND_WORD = "redo";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Redo undone command"
+            + "Example: " + COMMAND_WORD;
+
+    public static final String MESSAGE_SUCCESS = "Redo successfully";
```

Comment:
Maybe should change the wording to `Redo successful`, same for `Undo` -> `Undo successful`.

Reply:
Resolved at Commit 6bb04d6